crimsobot/cogs/games.py

```diff
@@ -598,6 +598,75 @@ async def daily(self, ctx: commands.Context, lucky_number: int) -> None:
 
         await ctx.send(embed=embed)
 
+    @commands.command(aliases=['bw'])
+    @commands.cooldown(5, 30, commands.BucketType.channel)
+    async def bubblewrap(self, ctx: commands.Context, *, bubble: str = None) -> None:
+        """Get a little sheet of bubble wrap to pop!
+        Provide any standard emoji or custom emoji from this server to pop!
+        If you use the command by itself, get a random emoji!
+        Short text inputs can also be used.
+
+        There is a per-channel cooldown on this command."""
+
+        def choose_random_emoji() -> str:
+            """Choose a random emoji from the emojis used in Emojistory."""
+            # read in lines of emojis
+            emojis = open(c.clib_path_join('games', 'emojilist.txt'), encoding='utf8', errors='ignore').read()
+            # strip newlines
+            emojis = emojis.replace('\n', '')
+            # choose and return
+            return random.choice(emojis)
```

Comment:
i feel like this function being _here_ is weird but the current functionality isn't duplicated anywhere as of right now, so this is probably fine.
future refactor: `Emoji` class for getting random emoji and such?

Reply:
We can only re-use this functionality on one other place right now: `emojistory`. I guess we have to think about whether it's worth refactoring in the future as part of an `emojistory` refresh/revisit.